hc/settings.py

```diff
@@ -166,6 +166,9 @@
 TWILIO_AUTH_TOKEN = os.environ.get("TWILIO_AUTH_TOKEN")
 TWILIO_NUMBER = os.environ.get("TWILIO_NUMBER")
 
+#Telegram integration
+TELEGRAM_TOKEN = os.environ.get('TELEGRAM_TOKEN')
```

Comment:
Set a default for this as discussed earlier

Reply:
@EdwinKato why is the default required?

Reply:
It's good practice to set defaults for many reasons. A case in point is this very app whereby it has a number default settings that are put in the `settings.py` file. For cases where the local-settings file doesn't exist, your application will still run very well with the defaults. There are cases where your tests will also depend on these values. The tests would fail in environments that missing these values.